### PR TITLE
Added a paragraph on ckBTC securit

### PR DESCRIPTION
**Pull request by csepreghy**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: src/pages/ckbtc.tsx [hunk 1]
```diff
@@ -538,7 +538,35 @@ function CkbtcPage(): JSX.Element {
 
                 <div>
                   <h3 className="tw-heading-5 md:tw-heading-3 mb-6">
-                    How it differs from Lightning.
+                    How ckBTC's security was assessed
+                  </h3>
+
+                  <p className="tw-paragraph mb-3">
+                    Several layers of security mesasures are taken on critical components of
+                    the Internet Computer such as ckBTC to ensure robust security. These
```
Comment (LaAlSc) [inline src/pages/ckbtc.tsx:546]:
```suggestion
                    Several security assessments are taken on critical components of
                    the Internet Computer such as ckBTC to ensure robust security. These
```
I think "security measures" refers more to design / code that prevents security risks vs. here we mean "we assessed if the code is secure" in teh examples

File: src/pages/ckbtc.tsx [hunk 2]
```diff
@@ -538,7 +538,35 @@ function CkbtcPage(): JSX.Element {
...
+                    include{" "}
+                    <Link href="https://medium.com/dfinity/eliminating-smart-contract-bugs-with-tla-e986aeb6da24/">
+                      TLA+ models
+                    </Link> to provide formal verification, several internal
```
Comment (LaAlSc) [inline src/pages/ckbtc.tsx:550]:
```suggestion
                    </Link> to formally verify some guarantees, several internal
```

File: src/pages/ckbtc.tsx [hunk 3]
```diff
@@ -538,7 +538,35 @@ function CkbtcPage(): JSX.Element {
...
+                    and external security assessments. In 2023, both Bitcoin integration and
+                    ckBTC have undergone a security audit conducted by{" "}
```
Comment (LaAlSc) [inline src/pages/ckbtc.tsx:552]:
```suggestion
                    ckBTC have undergone an external security audit conducted by{" "}
```

File: src/pages/ckbtc.tsx (hunk 2 above)
Comment (oggy-dfin) [inline src/pages/ckbtc.tsx:550]:
```suggestion
                    </Link> to provide formal verification, and several internal
```

**Review by artkorotkikh-dfinity — APPROVED** [review]

**Pull request merged**